kimchi/src/circuits/lookup/tables/bits16.rs

```diff
@@ -0,0 +1,27 @@
+use crate::circuits::lookup::tables::LookupTable;
+use ark_ff::Field;
+
+use super::BITS16_TABLE_ID;
+
+//~ The lookup table for 16-bits
+
+/// Returns the lookup table for all 16-bit values
+///
+/// # Panics
+///
+/// Will panic if `data` is invalid.
```

Comment:
Can this panic? I think you can remove this line, `data` is entirely internal

Reply:
Oh thanks. I believe this comment comes from a copy-paste of another file though.